src/proto/single/mutation.rs

```diff
@@ -0,0 +1,112 @@
+use crate::JobId;
+use derive_builder::Builder;
+
+#[cfg(doc)]
+use crate::{Client, Job};
+
+/// Mutation target set.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize)]
+#[serde(rename_all = "lowercase")]
+#[non_exhaustive]
+pub enum MutationTarget {
+    /// A set of currently enqueued jobs.
+    #[default]
+    Scheduled,
+
+    /// A set of jobs that should be retried.
+    Retries,
+
+    /// A set of failed jobs that will not be retried.
+    Dead,
+}
+
+// As of Faktory v1.9.2, not all the fields on the filter
+// will be taken into account, rather EITHER `jids` OR optional `kind`
+// plus optional `pattern`.
+// See: https://github.com/contribsys/faktory/issues/489
+//
+/// A filter to help narrow down the mutation target.
+///
+/// As of Faktory version 1.9.2, if [`MutationFilter::pattern`] and/or [`MutationFilter::kind`]
+/// is specified, the values in [`MutationFilter::jids`] will not be taken into account by the
+/// server. If you want to filter by `jids`, make sure to leave other fields of the filter empty
+/// or use dedicated methods like [`Client::requeue_by_ids`].
```

Comment:
If that's the case, I wonder if we shouldn't derive builder and instead just have specific constructors (for now) for the combinations that _do_ work. What do you think?